gui/default/syncthing/core/syncthingController.js

```diff
@@ -550,9 +568,6 @@ angular.module('syncthing.core')
 
                 data = data.connections;
                 for (id in data) {
-                    if (!data.hasOwnProperty(id)) {
-                        continue;
-                    }
```

Comment:
Why is this removed?

Reply:
Sorry I thought it was added as part of the PR, so took it along while refactoring.